### ISSUE TITLE
Totally ludicrous deluxetable error

### ISSUE THREAD

**Issue by augustfly** [AASJournals/AASTeX60#20]
The following code will throw a `! Missing number, treated as zero.` error, but if you swap the \tablehead line with the one commented out below so that the colhead value doesn't contain an "h" or an "H" then it will compile. This is the most baffling LaTeX bug I've ever seen.  

``` latex
\documentclass{aastex6}
\begin{document}
\title{Demo}
\author{Me}
\begin{abstract} Hello \end{abstract}


\begin{deluxetable}{crrrlcl}
\tablecolumns{7}
\tablecaption{ludicrous error}
\tablehead{\colhead{Width} & \colhead{H} & \colhead{R} & \colhead{D} & \colhead{E} & \colhead{F} & \colhead{G}}
%\tablehead{\colhead{W} & \colhead{X} & \colhead{R} & \colhead{D} & \colhead{E} & \colhead{F} & \colhead{G}}
\startdata
3 & 4 & 5 & 6 & 7 & 8 & 9 \\
3 & 4 & 5 & 6 & 7 & 8 & 9 \\
3 & 4 & 5 & 6 & 7 & 8 & 9 \\
\enddata
\tablecomments{}
\end{deluxetable}

\end{document}
```

**Comment by gregschwarz** [comment]
Actually I believe the problem is the "\colhead{Width}" bit which for some reason latex is treating as a the regular width command and thus is looking for a length to go with it. This is a known bug and will be addressed in the next release. I agree it is very odd.

**Comment by augustfly** [comment]
fwiw \colhead{Height} throws exactly the same problem.

**Comment by iancze** [comment]
I just experienced the same problem with `\colhead{Depth}` as well.

**Comment by augustfly** [comment]
@iancze thanks, and our apologies. the bug fix for these errors is forthcoming this summer.

**Comment by iancze** [comment]
@augustfly thanks for keeping the package updated! I think it's comical that we've all separately found this bug by appropriating the different dimensions of a rectangular prism as column headings.

**Comment by augustfly** [comment]
@iancze but did you find this issue by googling 'Totally ludicrous deluxetable error'? that would be hilarious... :-)

**Comment by dfm** [comment]
This _is_ ludicrous! I just hit it too. I find that `\colhead\ depth` works but then the header isn't centered. Is there a better temporary fix that I can use? I really want a column called "depth"!

**Comment by augustfly** [comment]
@dfm your workaround made me think of other classic latex workarounds that I just tried and appeared to me to work including:

```latex
\colhead{{W}idth}
\colhead{{D}epth}
\colhead{{H}eight}

\colhead{{width}}
\colhead{{depth}}
\colhead{{height}}
```

let me know if they work for you.

**Comment by dfm** [comment]
Beautiful. Yes that works (I used `{d}epth`). Thanks!

**Comment by augustfly** [comment]
I have tested and confirmed that this bug is fixed in v6.1. Test code below.

``` latex
\documentclass{aastex61}
\begin{document}
\title{Demo}
\author{Me}
\begin{abstract} Hello \end{abstract}


\begin{deluxetable}{crrrlcl}
\tablecolumns{7}
\tablecaption{ludicrous error}
\tablehead{\colhead{Width} & \colhead{height} & \colhead{Right} & \colhead{depth} & \colhead{Left} & \colhead{skip} & \colhead{GOTO}}
\startdata
3 & 4 & 5 & 6 & 7 & 8 & 9 \\
3 & 4 & 5 & 6 & 7 & 8 & 9 \\
3 & 4 & 5 & 6 & 7 & 8 & 9 \\
\enddata
\tablecomments{This is a trailing note}
\end{deluxetable}

\end{document}

```
